src/actions/publish.js

```diff
@@ -292,6 +297,11 @@ export function attemptToPublish(): ThunkAction<Promise<boolean>> {
             prePublishedState
           )
         );
+
+        // At this moment, we don't have the profile data in state anymore.
```

Comment:
Thought: This is very odd to me, even though I probably wrote this flow.

Suggestion: Is there a reason why it's the case, or is it just incidental to the current implementation? It would be nice to add some clarity as to why this happens.

Reply:
This _was_ very odd to me too, that's why I wrote this comment so that I don't waste so much time again next time :-)

What happens in short is that, when we publish a sanitized profile, we reset the profile-view state, which contains the profile data. It could make sense to do otherwise (for example move the profile data in another reducer, or keep the profile state when resetting the state). I didn't really want to invest more time thinking about that at that moment, as the solution of making MetaInfoPanel connected sounded like a good solution to me :-)

Reply:
I wrote this comment:
```
        // At this moment, we don't have the profile data in state anymore,
        // because the action profileSanitized will reset all the state of
        // profile-view, including the profile data. In the future we may want
        // to fix this (for example move the profile data in another reducer, or
        // keep the profile state when resetting the state).
        // This still works because it also sets the "phase" state to
        // "TRANSITIONING_FROM_STALE_PROFILE" that avoids rendering anything
        // (see AppViewRouter).
        // viewProfile below needs to synchronously dispatch the new profile
        // again so that the user doesn't see a glitch. This is still most
        // probably a performance problem because all components are unmounted
        // and mounted again.
```

Reply:
I wonder if that should be an issue 😅